src/components/LoginForm.jsx

```diff
@@ -0,0 +1,153 @@
+  const [password, setPassword] = useState("");
+  const [passwordVerification, setPasswordVerification] = useState("");
+  const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(
+    passwordMatchingText.valid
+  );
+
+  const handleTabChange = (event, tabIndex) => {
+    setCurrentTabIndex(tabIndex);
+    setEmail("");
+    setPassword("");
+    setPasswordVerification("");
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const handlePasswordChange = (event) => {
+    setPassword(event.target.value);
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  const handlePasswordVerificationChange = (event) => {
+    setPasswordVerification(event.target.value);
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+  const onLogin = (event) => {
+    event.preventDefault();
+    alert("Welcome " + email);
+  };
+
+  /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+  const onCreateAccount = (event) => {
+    event.preventDefault();
+    if (password !== passwordVerification) {
+      setPasswordMatchingMsg(passwordMatchingText.error);
+    }
+  };
+
+  return (
+    <Container
+      sx={{
+        width: "25%",
+        border: "1px solid gray",
+        minWidth: "200px",
+      }}
+    >
+      <Tabs
+        value={currentTabIndex}
+        sx={{ borderBottom: 1 }}
+        onChange={handleTabChange}
+      >
+        <Tab label="Login" />
+        <Tab label="Sign Up" />
+      </Tabs>
+      {currentTabIndex === 0 && (
+        <form onSubmit={onLogin}>
+          <Box sx={{ p: 3 }}>
+            <Stack spacing={2}>
+              <TextField
+                onChange={handleEmailChange}
+                value={email}
+                type="email"
+                id="email-login"
+                label="Email"
+                variant="outlined"
+              />
+              <TextField
+                onChange={handlePasswordChange}
+                value={password}
+                type="password"
+                id="password-login"
+                label="Password"
+                variant="outlined"
+                helperText=""
+              />
+              <Button
+                variant="contained"
+                type="submit"
+                sx={{ width: "25%", float: "right" }}
+              >
+                Login
+              </Button>
+            </Stack>
+          </Box>
+        </form>
+      )}
+      {currentTabIndex === 1 && (
+        <form onSubmit={onCreateAccount}>
+          <Box sx={{ p: 3 }}>
+            <Stack spacing={2}>
+              <TextField
+                onChange={handleEmailChange}
+                value={email}
+                type="email"
+                id="email-signup"
+                label="Email"
+                variant="outlined"
+              />
```

Comment:
Don't need to change anything now, but I think eventually we'll want to change "password" to something like "master password" and add some sort of instructions/warning. Here's the password fields from BitWarden's signup form as an example:

<img width="503" alt="Screenshot 2023-10-30 at 9 16 24 AM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/1830adf3-3bea-459e-82fa-71a277e89e8b">

<img width="499" alt="Screenshot 2023-10-30 at 9 15 49 AM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/a39897a6-ce1c-4e6c-8504-cfc7c4edae7f">